starport/pkg/cosmosutil/path.go

```diff
@@ -0,0 +1,71 @@
+package cosmosutil
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+)
+
+// ChainHomeRoot is the root dir for spn chain homes
+const (
+	ChainHomeRoot = "spn"
+	gentxPath     = "config/gentx/gentx.json"
+	genesisPath   = "config/genesis.json"
+)
+
+// IsChainHomeExist checks if a home with the provided launchID already exist
+func IsChainHomeExist(launchID uint64) (string, bool, error) {
+	home, err := ChainHome(launchID)
+	if err != nil {
+		return home, false, err
+	}
+
+	if _, err := os.Stat(home); os.IsNotExist(err) {
+		return home, false, nil
+	}
+	return home, true, err
+}
+
+// ChainHome returns the default home dir used for a chain from SPN
+func ChainHome(launchID uint64) (string, error) {
+	home, err := os.UserHomeDir()
+	return filepath.Join(home, ChainHomeRoot, strconv.FormatUint(launchID, 10)), err
+}
```

Comment:
I think This part should remain in `network` package since it's related to `spn`